pkg/sql/tablemetadatacache/table_metadata_updater.go

```diff
@@ -40,38 +45,83 @@ type tableMetadataDetails struct {
 	StatsLastUpdated *time.Time `json:"stats_last_updated"`
 }
 
-// newTableMetadataUpdater creates a new tableMetadataUpdater.
-func newTableMetadataUpdater(ie isql.Executor) *tableMetadataUpdater {
-	return &tableMetadataUpdater{ie: ie, upsertQuery: newTableMetadataBatchUpsertQuery(tableBatchSize)}
+var _ tablemetadatacacheutil.ITableMetadataUpdater = &tableMetadataUpdater{}
+
+// newTableMetadataUpdater creates a new iTableMetadataUpdater.
```

Comment:
Can we update the comment here to remove `ITableMetadataUpdater`.

Reply:
Done